### ISSUE TITLE
doesn't work for chrome

### ISSUE THREAD

**Issue by metapea** [FireMasterK/BypassAdditions#8]
this works fine in firefox but not on chrome.
Do you mind adding basic support for chrome?

**Comment by Lucas559-noob** [comment]
if it work on edge(based in chromium(chrome is based in it too)) it should work on chrome too
maybe is your monkey that is not working, try changing to violentmonkey

**Comment by metapea** [comment]
Nope, i already tested violentmonkey on firefox and chromium. It still says "Complete the following steps" in chrome.

**Comment by Nothing4You** [comment]
is this still an issue? works fine for me in chrome 95 using violentmonkey.

**Comment by metapea** [comment]
It's fine so far, but when i clicked on one of the links with the url for the first time it keep loading on a blank webpage with the url until i refreshed then it worked. Although the rest is working
Edit: the same link is working fine now, i will keep this up for some time
